liquid.py

```diff
@@ -285,9 +352,83 @@ def get_job(hcl_path, substitutions={}):
     return template.safe_substitute(set_volumes_paths(substitutions))
 
 
+def gc():
+    """Stop collections jobs that are no longer declared in the ini file."""
+
+    collections = dict(config.collections)
+    nomad_jobs = nomad.jobs()
+
+    for job in nomad_jobs:
+        if job['ID'].startswith('collection-'):
+            collection_name = job['ID'][len('collection-'):]
+            if collection_name not in collections:
+                log.info('Stopping %s...', job['ID'])
+                nomad.stop(job['ID'])
+
+
+def get_collections_to_purge():
+    """Returns a set of collections to purge."""
+
+    ini_collections = set(dict(config.collections))
```

Comment:
Maybe `config.collections` should be an [OrderedDict](https://docs.python.org/3/library/collections.html#collections.OrderedDict) at this point.